src/components/emby-button/emby-button.js

```diff
@@ -21,30 +21,23 @@ define(['browser', 'dom', 'layoutManager', 'shell', 'appRouter', 'apphost', 'css
     }
 
     EmbyButtonPrototype.createdCallback = function () {
-
         if (this.classList.contains('emby-button')) {
             return;
         }
 
         this.classList.add('emby-button');
-
         if (layoutManager.tv) {
-            if (this.getAttribute('data-focusscale') !== 'false') {
-                this.classList.add('emby-button-focusscale');
-            }
-            this.classList.add('emby-button-tv');
+            // handles all special css for tv layout
+            // the old method was element-showfocus
+            // this method utilizes class chaining instead
```

Comment:
no use leaving comment like "old method was" - we no longer see it in the code, so there's no "old method". probably something like
```suggestion
            // handles all special css for tv layout;
            // this method utilizes css class chaining
```